js/build.js

```diff
@@ -30,6 +30,12 @@ Fliplet.Widget.instance('list-thumb-l', function(data) {
     }
   });
 
+  $container.on('click', '.filter-' + data.id, function(event) {
```

Comment:
You're building a feature that the `SwipeSaveList` class is meant to have implemented already.

https://github.com/Fliplet/fliplet-widget-list-large-thumbs/blob/663945a7d0819c871b9432c9c60f4757c164a022/js/build.js#L40

Please investigate and explain why this isn't working before developing a new implementation.

Reply:
In `SwipeSaveList` we add `.saved` class to list items
![image](https://user-images.githubusercontent.com/74735207/104310077-cc793a80-54db-11eb-8e67-220519cad8c8.png)
but on changing the list name, we add '.my' only to '.label-instructions'
![image](https://user-images.githubusercontent.com/74735207/104311565-0c412180-54de-11eb-9d43-6b5af2b67e95.png)
I think it would be better to add '.my' also to '.list-holder'.

Reply:
I added the `my` class to the `list-holder` container and that didn't seem to help. See highlighted in green below.

![image](https://user-images.githubusercontent.com/290733/104337871-d7d56180-54ed-11eb-88d2-e15cf256f2ba.png)

I also notice that I'm seeing the **Remove from My List** boxes appearing to the right too.

This feature worked before, and I don't think your investigation explains why it isn't working any more.

I've added an example of this in the app **ListApp** and screen **Large thumb list** in your account.

Reply:
I think I've found the problem, we need to remove `display flex;` which was added in that commit https://github.com/Fliplet/fliplet-widget-list-large-thumbs/commit/c6bc9dbbd9f3ae02ebb0d955b86b29884d999155